pkg/ccl/benchccl/rttanalysisccl/rttanalysisccl_test: TestBenchmarkExpectation failed

pkg/ccl/benchccl/rttanalysisccl/rttanalysisccl_test.TestBenchmarkExpectation [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20397755?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20397755?buildTab=artifacts#/) on master @ [9e30767e7a4c6b3e9d8232a2edbd0dc2d835f34c](https://github.com/cockroachdb/cockroach/commits/9e30767e7a4c6b3e9d8232a2edbd0dc2d835f34c):


```
=== RUN   TestBenchmarkExpectation
    validate_benchmark_data.go:186: [/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/9030/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/benchccl/rttanalysisccl/rttanalysisccl_test_/rttanalysisccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/benchccl/rttanalysisccl/rttanalysisccl_test_/rttanalysisccl_test --test.v=test2json --test.bench=^$]
I250827 17:25:26.090567 1 (gostd) rand.go:268  [-] 1  random seed: 8720407500515141612
=== RUN   TestBenchmarkExpectation
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/7418df5abfb13dbfaf6c0beee25f240a/logTestBenchmarkExpectation3963028469
    test_log_scope.go:82: use -show-logs to present logs inline
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_regional_by_table_to_global to perform KV lookups in [13, 13], got 13
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_global_to_regional_by_table to perform KV lookups in [13, 13], got 13
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_rbr_to_global to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_populated_database_alter_primary_region to perform KV lookups in [12, 14], got 12
    validate_benchmark_data.go:120: success: expected AlterRegions/alter_populated_database_add_region to perform KV lookups in [11, 11], got 11
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_populated_database_from_region_to_zone to perform KV lookups in [22, 22], got 22
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_empty_database_set_initial_primary_region to perform KV lookups in [13, 13], got 13
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_global_to_rbr to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_empty_database_alter_primary_region to perform KV lookups in [12, 14], got 12
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_rbr_to_regional_by_table to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterTableLocality/alter_from_regional_by_table_to_rbr to perform KV lookups in [18, 18], got 18
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_empty_database_from_zone_to_region to perform KV lookups in [12, 12], got 12
    validate_benchmark_data.go:120: success: expected AlterPrimaryRegion/alter_populated_database_set_initial_primary_region to perform KV lookups in [14, 14], got 14
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_from_crdb_internal.zones_(100_tables) to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_empty_database_from_region_to_zone to perform KV lookups in [12, 12], got 12
    validate_benchmark_data.go:120: success: expected AlterRegions/alter_empty_database_drop_region to perform KV lookups in [12, 16], got 14
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_from_crdb_internal.zones_(10_tables) to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterSurvivalGoals/alter_populated_database_from_zone_to_region to perform KV lookups in [22, 22], got 22
    validate_benchmark_data.go:120: success: expected VirtualTableQueries/select_from_crdb_internal.zones_(50_tables) to perform KV lookups in [10, 10], got 10
    validate_benchmark_data.go:120: success: expected AlterRegions/alter_populated_database_drop_region to perform KV lookups in [14, 14], got 14
    validate_benchmark_data.go:73: see the -rewrite flag to re-run the benchmarks and adjust the expectations
    validate_benchmark_data.go:74: usage: ./dev test --rewrite pkg/bench/rttanalysis -f=TestBenchmarkExpectation/ --test-args '-rewrite-iterations=N'
    validate_benchmark_data.go:107: -- test log scope end --
I250827 17:26:29.834290 682100 sql/catalog/lease/descriptor_state.go:238  [T1,Vsystem,n2,intExec=create-stats] 2  release: 15("jobs",010180bf9e06185de54beca8cc8272e5ed8bb3) ver=1:0,0, refcount=0
I250827 17:26:29.834368 682100 sql/catalog/lease/descriptor_state.go:238  [T1,Vsystem,n2,intExec=create-stats] 3  release: 54("job_info",010180bf9e06185de54beca8cc8272e5ed8bb3) ver=1:0,0, refcount=0
test logs left over in: /artifacts/tmp/_tmp/7418df5abfb13dbfaf6c0beee25f240a/logTestBenchmarkExpectation3963028469
--- FAIL: TestBenchmarkExpectation (63.74s)
--- FAIL: TestBenchmarkExpectation (64.20s)
=== RUN   TestBenchmarkExpectation/AlterRegions
--- FAIL: TestBenchmarkExpectation/AlterRegions (63.72s)
=== RUN   TestBenchmarkExpectation/AlterRegions/alter_empty_database_add_region
    multi_region_bench_test.go:34: pq: setting updated but timed out waiting to read new value
--- FAIL: TestBenchmarkExpectation/AlterRegions/alter_empty_database_add_region (19.91s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBenchmarkExpectation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-53942

--- a/pkg/ccl/benchccl/rttanalysisccl/multi_region_bench_test.go
+++ b/pkg/ccl/benchccl/rttanalysisccl/multi_region_bench_test.go
@@ -28,6 +28,9 @@ var reg = rttanalysis.NewRegistry(numNodes, rttanalysis.MakeClusterConstructor(f
 	cluster, _, cleanup := multiregionccltestutils.TestingCreateMultiRegionCluster(
 		tb, numNodes, knobs,
 	)
+	if err := cluster.WaitForFullReplication(); err != nil {
+		tb.Fatal(err)
+	}
 	db := cluster.ServerConn(0)
 	// Eventlog is async, and introduces jitter in the benchmark.
 	if _, err := db.Exec("SET CLUSTER SETTING server.eventlog.enabled = false"); err != nil {

